### PR TITLE
Use "echo" binary, not the built-in command

### PR DESCRIPTION
**Pull request by msrb**
Built-in "echo -n" doesn't work on Mac(?)

### CODE REVIEW

File: openshift/helpers.sh [hunk 1]
```diff
@@ -29,15 +29,15 @@ function generate_and_deploy_config() {
...
-    -p RDS_PASSWORD="$(echo -n "${RDS_PASSWORD}" | base64)" \
+    oc process -p AWS_ACCESS_KEY_ID="$(/bin/echo -n "${AWS_ACCESS_KEY_ID}" | base64)" \
+    -p AWS_SECRET_ACCESS_KEY="$(/bin/echo -n "${AWS_SECRET_ACCESS_KEY}" | base64)" \
+    -p GITHUB_API_TOKENS="$(/bin/echo -n "${GITHUB_API_TOKENS}" | base64)" \
+    -p GITHUB_OAUTH_CONSUMER_KEY="$(/bin/echo -n "${GITHUB_OAUTH_CONSUMER_KEY}" | base64)" \
+    -p GITHUB_OAUTH_CONSUMER_SECRET="$(/bin/echo -n "${GITHUB_OAUTH_CONSUMER_SECRET}" | base64)" \
+    -p LIBRARIES_IO_TOKEN="$(/bin/echo -n "${LIBRARIES_IO_TOKEN}" | base64)" \
+    -p FLASK_APP_SECRET_KEY="$(/bin/echo -n "${FLASK_APP_SECRET_KEY}" | base64)" \
+    -p RDS_ENDPOINT="$(/bin/echo -n "${RDS_ENDPOINT}" | base64)" \
+    -p RDS_PASSWORD="$(/bin/echo -n "${RDS_PASSWORD}" | base64)" \
```
Comment (geetikabatra) [inline openshift/helpers.sh:40]: @msrb Looks good, should we add `RDS_DBNAME` also here.

Comment (msrb) [reply openshift/helpers.sh:40]: @GeetikaBatra we don't have a parameter for that in `secrets-template.yaml`, we only use it when creating RDS instances with the deploy.sh script.

**Comment by geetikabatra** [comment]
This looks good to me. Can we merge it. Thanks in advance. :)

**Comment by geetikabatra** [comment]
Merging it :)

**Pull request merged**